common.mk

```diff
@@ -21,12 +21,6 @@ CLINK ?=
 ## as can be the case if python is set via an alias)
 PYTHON:=python
```

Comment:
Is this safe?  If I understood the first problem in #208 correctly, the issue was that `python` did not exist on the system while `python3` did.

So could we do `PYTHON ?= python` and then populate `PYTHON` from `setup.py`, maybe via `sys.executable`?

Reply:
You are spot on -- `setup.py` updates that line in `common.mk` with the `sys.executable` -- [here](https://github.com/manodeep/Corrfunc/blob/master/setup.py#L233-L242)